### PR TITLE
to_wiki_url in the RuntimeLibrary

### PR DESCRIPTION
**Pull request by gausie**
- Add tests for to_wiki_url function
- Write to_wiki_url functions

### CODE REVIEW

File: src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java [hunk 1]
```diff
@@ -3654,6 +3670,20 @@ public static Value to_url(ScriptRuntime controller, final Value value) {
         : new Value(adventure.getRequest().getURLString());
   }
 
+  public static Value to_wiki_url(ScriptRuntime controller, final Value value) {
+    var type =
+        switch (value.getType().getName()) {
```
Comment (midgleyc) [inline src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:3675]: can you not switch on `getType()` directly?

**Review by midgleyc — CHANGES_REQUESTED** [review]

File: src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java (hunk 1 above)
Comment (gausie) [reply src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:3675]: No, switches have to be primitives or enums!

Comment (gausie) [reply src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:3675]: And before you go in and try to enum types... the issue is that we use the same class to make arbitrary types at runtime

Comment (midgleyc) [reply src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:3675]: I thought `getType` was a TypeSpec.

Comment (midgleyc) [reply src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:3675]: okay, that's `value.getType().getType()`. Can you switch on that instead for slightly less stringly-typed code?

Comment (gausie) [reply src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:3675]: yeah ok

**Review by midgleyc — APPROVED** [review]
Nice :D

**Pull request merged**